modules/ROOT/pages/configuration.adoc

```diff
@@ -77,16 +77,23 @@ documentation on all possible configuration options for Neo4j Drivers, see the l
 4+|*Driver options*
 
 |`url`
-|The url of the Neo4j instance to connect to.
-You can provide a comma-separated list of valid Neo4j URIs like: `neo4j://localhost,neo4j://foo.bar,neo4j://foo.bar.baz:7783`, the first one
-will be used as base URL and the others as resolvers just as explained
-link:https://neo4j.com/docs/java-manual/current/client-applications/#java-driver-resolver-function[here, window=_blank]
+a|The url of the Neo4j instance to connect to.
+
+When provided with a comma-separated list of URIs, link:[Resolver function] feature of the driver will be activated, and the first URI will be used as original host while the rest are treated as resolver function outputs.
 |_(none)_
 |Yes
 
 |`authentication.type`
-|The authentication methods to be used: `none`, `basic`, `kerberos`, `custom`, `bearer`.
-More information link:https://neo4j.com/docs/java-manual/4.4/client-applications/#java-driver-authentication[here, window=_blank]
+a|
+The authentication methods to be used:
+
+* `none`
+* `basic`
+* `kerberos`
+* `custom`
+* `bearer` label:new[Introduced in 5.1]
```

Comment:
Do you think if we make it shorter like this, it's still understandable? It just looks better on the website

```suggestion
* `bearer` label:new[v.5.1]
```